test/serena/util/test_project_gather_source_files.py

```diff
@@ -0,0 +1,25 @@
+from serena.project import Project
+from solidlsp.ls_config import Language
+
+
+def test_gather_source_files_includes_language_sources(tmp_path):
```

Comment:
I don't think this tests adds anything. We have integration tests that read language files, I propose to remove it.